src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -4310,4 +4310,7 @@
   <data name="WasmThreads_BlockingWaitNotSupportedOnJSInterop" xml:space="preserve">
     <value>Blocking wait is not supported on the JS interop threads.</value>
   </data>
-</root>
+  <data name="Arg_MustBeBFloat16" xml:space="preserve">
+    <value>Object must be of type BFloat16.</value>
+  </data>
```

Comment:
nit: I think we try to keep these mostly alphabetical, as that's what editing via the IDE will do.

Reply:
I literally added it via the IDE, and it's added to the end.